pootle/apps/pootle_store/unit/timeline.py

```diff
@@ -60,24 +62,46 @@ def gravatar_url(self, size=80):
 
 class SuggestionEvent(object):
 
-    def __init__(self, submission_type, username, full_name):
+    def __init__(self, submission_type, username, full_name, comment):
         self.submission_type = submission_type
         self.username = username
         self.full_name = full_name
+        self.comment = comment
 
     @cached_property
     def user(self):
         return DisplayUser(self.username, self.full_name)
 
     @property
     def description(self):
-        author_link = self.user.author_link
```

Comment:
actually im not sure if this will work properly in terms of gettext. just trying to think how to make this a bit tidier